common/src/services/export.service.ts

```diff
@@ -39,22 +42,22 @@ export class ExportService implements ExportServiceAbstraction {
     private cryptoFunctionService: CryptoFunctionService
   ) {}
 
-  async getExport(format: "csv" | "json" | "encrypted_json" = "csv"): Promise<string> {
+  async getExport(format: ExportFormat = "csv", organizationId?: string): Promise<string> {
+    if (organizationId) {
```

Comment:
Why? are we not using falsiness anymore?

Reply:
I thought it was our code style, and is definitely my preference because it's clearer what you're testing (false? null/undefined? empty string? etc). But if I'm mistaken here that's fine.